extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -89,6 +93,11 @@ export class AmpAnalytics extends AMP.BaseElement {
     this.requests_ = {};
 
     /**
+     * @private {Object<string, RequestHandler>} A map of request handler with requests-v2
+     */
+    this.requestsV2_ = {};
```

Comment:
I'd rather we drop "v2" suffix and merge this structure into `this.requests_`.

Reply:
done